client/src/layout/Layout/Layout.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { Outlet } from 'react-router-dom';
+import Footer from '../Footer/Footer';
```

Comment:
we have in index.js folder in our layout. please use it as per our agreement to export all needed components from there.  then you can just import your Footer like below:

```suggestion
import Footer from '../layout';
```